blog/2023-10-24-monitoring-vs-observability/index.md

```diff
@@ -0,0 +1,138 @@
+---
+slug: monitoring-vs-observability
+title: "Monitoring vs Observability: What Engineers Need to Know"
+description: |
+  Explore the key differences and synergies between monitoring and observability, and discover how to leverage each for optimal system performance in today's complex, data-rich environments.
+authors: shyam
+tags: [monitoring, observability, engineers, devops, modern, systems]
+keywords: [monitoring, observability, engineers, devops, modern, systems]
+image: https://github.com/netdata/blog/assets/24860547/bc45acdc-c32d-430e-b477-aa8c0734d229
+---
+
+![blog](https://github.com/netdata/blog/assets/24860547/bc45acdc-c32d-430e-b477-aa8c0734d229)
+
+As systems increasingly shift towards distributed architectures to deliver application services, the roles of monitoring and observability have never been more crucial. Monitoring delivers the situational awareness you need to detect issues, while observability goes a step further, offering the analytical depth to understand the root cause of those issues.
+
+Understanding the nuanced differences between monitoring and observability is crucial for anyone responsible for system health and performance. In dissecting these methodologies, we'll explore their unique strengths, dive into practical applications, and illuminate how to strategically employ each to enhance operational outcomes.
+
+To set the stage, consider a real-world scenario that many of us have encountered: It's 3 a.m., and you get an alert that a critical service is down. Traditional monitoring tools may tell you what's wrong, but they won't necessarily tell you why it's happening leaving that part up to you. With Observability, the tool enables you to explore your system's internal state and uncover the root cause in a faster and easier manner. 
```

Comment:
```suggestion
To set the stage, consider a real-world scenario that many of us have encountered: It's 3 a.m., and you get an alert that a critical service is down. Traditional monitoring tools may tell you what's wrong, but they won't necessarily tell you why it's happening leaving that part up to you. With observability, the tool enables you to explore your system's internal state and uncover the root cause in a faster and easier manner. 
```

throughout the document you use only caps on titles, so suggested change for consistency